### PR TITLE
Add check environ.get DISPLAY

### PR DESCRIPTION
**Pull request by patsevanton**

### CODE REVIEW

File: matrix/matrix.py [hunk 1]
```diff
@@ -394,6 +396,10 @@ def skydive_output(matrix):
 
 def main():
 
+    if environ.get('DISPLAY') is None:
+        print("skydive-flow-matrix need run on pc with GUI")
```
Comment (safchain) [inline matrix/matrix.py]: is it still true is we choose another engine or output format ?

Comment (patsevanton) [reply matrix/matrix.py]: Hmm. may be relocate check when run render?

Comment (safchain) [reply matrix/matrix.py]: yes sounds good, thx

Comment (patsevanton) [reply matrix/matrix.py]: relocated

**Pull request merged**